release_util_script_encrypt_var.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Ditto - this script isn't referred to in documentation so what is it needed for?

Reply:
Added to file:
```
# This script can be used to encrypt a variable (e.g. gpg key id, if a key is
# provided by someone else)
# NB release_util_script_create_gpg provides both the key id and also its
# encrypted form.
# This script is the peer of 'release_util_script_decrypt_var.sh' - which can be used to
# double check that an encrypted variable (e.g. gpg key id) is correct (e.g. no
# misspellings have occurred).
```